AsyncDisplayKit/ASTextNode.mm

```diff
@@ -48,30 +42,16 @@ - (instancetype)initWithRenderer:(ASTextKitRenderer *)renderer
 
 @implementation ASTextNodeDrawParameters
 
-- (instancetype)initWithRenderer:(ASTextKitRenderer *)renderer
-                      textOrigin:(CGPoint)textOrigin
-                 backgroundColor:(UIColor *)backgroundColor
+- (instancetype)initWithBounds:(CGRect)bounds
+               backgroundColor:(UIColor *)backgroundColor
 {
   if (self = [super init]) {
-    _renderer = renderer;
-    _textOrigin = textOrigin;
+    _bounds = bounds;
     _backgroundColor = backgroundColor;
   }
   return self;
 }
 
-- (void)dealloc
```

Comment:
Looks like this may have been needed on the TextNode itself instead / or as well. I'll move it there.